Extensions/WikiUpdater/WikiUpdaterTask/src/GitWikiFuntions.ts

```diff
@@ -90,7 +90,12 @@ export async function UpdateGitWikiFile(repo, localpath, user, password, name, e
         }
         logInfo(`Cleaned ${localpath}`);
 
-        await git.silent(true).clone(remote, localpath);
+        var extraHeaders = [];
+        if (injectExtraHeader) {
+            extraHeaders = [`-c http.extraheader="AUTHORIZATION: bearer ${password}"`];
```

Comment:
remote should be `https://${repo}` or else this won't work

Reply:
Actually, I have just run the test. It does work even with the remote that contains user:password. My bad.

Reply:
Thanks for the feedback. I will revise the PR

Reply:
Is the last commit nearer that you think is required